steps/tuist/0.1.0/step.yml

```diff
@@ -0,0 +1,44 @@
+title: tuist
+summary: |
+  Run Tuist from Bitrise's CI environment
+description: |
+  Run Tuist from Bitrise's CI environment
```

Comment:
This text should be more descriptive. What is a tuist? What problem does it try to solve etc.